### DISCUSSION TITLE
How can I get list of versions via REST endpoint in Directus interface?

### DISCUSSION THREAD

**Discussion by Zlite-147** [directus/directus#23717] [Feature Requests]
### Summary

How can I get list of versions via REST endpoint in Directus interface? 
Means I want to add dropdown of version available in system via REST Endpoint.
[
![Directus](https://github.com/user-attachments/assets/c7f3037e-71cc-44a7-af65-e0cf8491a034)
](url)

### Basic Example

Say an Example,

There is a content node called "Content1" in which I want create a version, But before creating version, We want dropdown of list of version available in System. To fetch version available in system, We have developed REST endpoint.

### Motivation

Managing versions effectively in an enterprise application is crucial. Fetching existing version names via the REST API can prevent duplicate entries in dropdowns, ensuring unique version names and reducing errors. This enhances the user experience and maintains data integrity.

### Detailed Design

1. Create an interface which allow dropdown values Via API

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

NA

### Alternatives

NA

### Adoption Strategy

NA

**Comment by br41nslug** [comment]
Content versions can be listed using this endpoint https://docs.directus.io/reference/system/versions.html#list-content-versions

**Reply by Zlite-147** [reply]
Yes, I have developed the API for fetching Content Version, But My concerns is that I want to pop up all available versions in a dropdown whenever user try to create new version.

**Reply by br41nslug** [reply]
Not sure im completely following, that dropdown already exists right? as shown in the screenshot

**Reply by Zlite-147** [reply]
This is by default dropdown, I want to add values to dropdown Via REST call.

**Reply by br41nslug** [reply]
Ah there is indeed currently no way to hook into that besides actually creating a new version using the API/App
